yt/data_objects/construction_data_containers.py

```diff
@@ -1290,35 +1290,51 @@ class YTSmoothedCoveringGrid(YTCoveringGrid):
     _min_level = None
 
     @wraps(YTCoveringGrid.__init__)
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, interp="linear", nbuf=None, **kwargs):
```

Comment:
so as to avoid any possible collision with positional arguments (`*args`), it may be worth making these new arguments keyword only :
```suggestion
    def __init__(self, *args, *, interp="linear", nbuf=None, **kwargs):
```

Reply:
Yep, that sounds good. I didn't realize that is how you did it.